pubsub/awssub_test.go

```diff
@@ -234,10 +234,10 @@ func (s *TestSQSAPI) GetQueueAttributesRequest(*sqs.GetQueueAttributesInput) (*r
 func (s *TestSQSAPI) GetQueueAttributes(*sqs.GetQueueAttributesInput) (*sqs.GetQueueAttributesOutput, error) {
 	return nil, errNotImpl
 }
-func (s *TestSQSAPI) GetQueueUrlRequest(*sqs.GetQueueUrlInput) (*request.Request, *sqs.GetQueueUrlOutput) {
+func (s *TestSQSAPI) GetQueueURLRequest(*sqs.GetQueueURLInput) (*request.Request, *sqs.GetQueueUrlOutput) {
```

Comment:
This makes golint pass, but actually breaks the code. AWS's library unfortunately generates names that do not match Go conventions :disappointed: 

https://godoc.org/github.com/aws/aws-sdk-go/service/sqs#GetQueueUrlInput